<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TiMagic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="deviceListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>218</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnAndroidDevices">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Install to connected Android Devices: </string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnAPK">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>99</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Google Play Store APK</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnIPA">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Ad-Hoc IPA</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeadline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HanziPenTC-W3</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TiMagic</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Project:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnIOS6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>iOS 6.1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioBtnIOS7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>iOS 7.0</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QToolButton" name="btnReloadProjects">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>44</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelDeviceCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>2</y>
       <width>31</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="btnReloadDeviceCount">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>44</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="consoleLogTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAPK_IPA_Magic">
    <property name="title">
     <string>TiMagic</string>
    </property>
    <addaction name="actionRun_current_Configuration"/>
    <addaction name="actionAbout"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuAPK_IPA_Magic"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRun_current_Configuration">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run the currently active configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtnIPA</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioBtnIOS6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtnIPA</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioBtnIOS7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
